src/cli.ts

```diff
@@ -96,14 +96,19 @@ addCommand('checkout', 'Check out any repositories that are candidates for a giv
 
 const applyCommand = buildCommand('apply', 'Apply a migration to all checked out repositories');
 addReposOption(applyCommand);
-// Commander's API is weird: this will get mapped to the option `resetOnError`, which will be false if this option
-// is specified. We want to default `resetOnError` to true here.
-applyCommand.option('--no-reset-on-error', 'Keep changes in the working tree even if the migration fails', true);
+applyCommand.option('--skip-reset-branch', 'Don\'t reset branch before applying', false);
+applyCommand.option('--force-reset-branch', 'Force a reset of the branch before applying', true);
```

Comment:
Maybe we should drop `branch` from these flags and just call them `--skip-reset` and `--force-reset`?

Reply:
I'm trying to differentiate between the `reset` that removes uncommitted changes, the `reset` that resets the branch back to `master`, and the final `reset` that happens by default if the migration fails. That's definitely an overloaded term and I'm hoping to make the reason for the name clear in the docs.